automerge/Cargo.toml

```diff
@@ -9,6 +9,7 @@ license = "MIT"
 [features]
 optree-visualisation = ["dot"]
 wasm = ["js-sys", "wasm-bindgen"]
+arbitrary = ["dep:arbitrary", "smol_str/arbitrary"]
```

Comment:
Note that this currently requires a nightly version of `cargo`, though it is being stabilised in the next rust release (1.60.0) https://github.com/rust-lang/rust/pull/94817 as namespaced-features.

Reply:
This is being released in two weeksright? Do we wait until then to merge the PR?

Reply:
I was thinking we probably could.